.gitignore

```diff
@@ -50,14 +44,8 @@ TAGS.*
 \#*\#
 config.mk
 config.stamp
-keywords.md
-lexer.ml
 Session.vim
 src/etc/dl
```

Comment:
No such dir exists here, but I wasn't sure if *something* wouldn't create it? I would argue that's a bug though, everything should happen in `build/`.

Reply:
This I think is pretty old and can be removed